src/storage/ExtensionBasedMapper.ts

```diff
@@ -135,13 +148,16 @@ export class ExtensionBasedMapper implements FileIdentifierMapper {
    */
   public async mapFilePathToUrl(filePath: string, isContainer: boolean): Promise<ResourceLink> {
     if (!filePath.startsWith(this.rootFilepath)) {
-      throw new Error(`File ${filePath} is not part of the file storage at ${this.rootFilepath}.`);
+      this.logger.error(`Trying to access file ${filePath} outside of ${this.rootFilepath}`);
+      throw new Error(`File ${filePath} is not part of the file storage at ${this.rootFilepath}`);
     }
 
     let relative = filePath.slice(this.rootFilepath.length);
     if (isContainer) {
+      const path = ensureTrailingSlash(this.baseRequestURI + encodeUriPathComponents(relative));
+      this.logger.verbose(`Container filepath ${filePath} maps to URL ${path}`);
```

Comment:
Why suddenly verbose instead of info?

Reply:
I thought that these functions might be called often and thus cause a lot of spam otherwise. I will change this to info if you feel okay with this.

Reply:
Not more than the other functions in the mapper, or `getAclRecursive` so would keep it the same.